index.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
+
+const ul = document.getElementById('ul-list');
+const form = document.getElementById('form');
+const inputAuthor = document.getElementById('author');
+const inputBook = document.getElementById('book');
+
+const bookDetails = JSON.parse(localStorage.getItem('newBooksAdded')) || [];
+
+function addBook() {
+  ul.innerHTML = '';
+  for (let i = bookDetails.length - 1; i >= 0; i -= 1) {
+    const bookInfo = bookDetails[i];
+    const newBook = document.createElement('li');
+    newBook.innerHTML = `
+      <p class="book">${bookInfo.book}</p>
+      <p class="author">${bookInfo.author}</p>
+      <button type="button" onclick="removeItem(${i})">Remove</button>
+      <hr>
+    `;
+    ul.insertBefore(newBook, ul.firstChild);
+  }
+}
+
+function removeItem(index) {
+  bookDetails.splice(index, 1);
+  addBook();
+  localStorage.setItem('newBooksAdded', JSON.stringify(bookDetails));
+}
+
+// remove item
+removeItem();
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (inputAuthor.value === '' || inputBook.value === '') {
+    return;
+  }
+  const newBook = { author: inputAuthor.value, book: inputBook.value };
+  bookDetails.unshift(newBook);
+  localStorage.setItem('NewBookAdded', JSON.stringify(bookDetails));
```

Comment:
- [ ] This should be `newBooksAdded` according to what you defined above.

- [ ] After fixing that, when I add three books and refresh one book is gone from the storage and the page, if I refresh again, another one is gone, until there is none. Kindly look into your code and ensure all added books are persisted correctly.

- [ ] When I refresh the page the added books are not displayed on the page.  Kindly ensure that when the books are added they are correctly added to the local storage.